dizoo/mujoco/envs/mujoco_disc_env.py

```diff
@@ -0,0 +1,167 @@
+import copy
+import os
+from itertools import product
+from typing import Union, List, Optional
+
+import gym
+import matplotlib.pyplot as plt
+import numpy as np
+from ding.envs import BaseEnv, BaseEnvTimestep
+from ding.envs.common.common_function import affine_transform
+from ding.torch_utils import to_ndarray
+from ding.utils import ENV_REGISTRY
+from easydict import EasyDict
+from matplotlib import animation
+
+from .mujoco_wrappers import wrap_mujoco
+
+
+@ENV_REGISTRY.register('mujoco-disc')
+class MujocoDiscEnv(BaseEnv):
+
+    @classmethod
+    def default_config(cls: type) -> EasyDict:
+        cfg = EasyDict(copy.deepcopy(cls.config))
+        cfg.cfg_type = cls.__name__ + 'Dict'
+        return cfg
+
+    config = dict(
+        use_act_scale=False,
```

Comment:
action is already in [-1, 1], remove `use_act_scale` in all mujoco envs and configs and add another `action_clip` flag to control whether to do action clip operation ([-1, 1]), defaults to False